assets/survey_themes/fruity_twentythree/typography/typography.scss

```diff
@@ -25,7 +25,6 @@ label {
 
 .large-heading {
   @include largeheading;
-  font-family: $ibm-serif-font-family;
```

Comment:
large headings should be in serif style as by designguide, so this needs to keep its serif font as opposed to the default font which is sans-serif (without serifs)